sympy/core/symbol.py

```diff
@@ -201,15 +201,43 @@ def numbered_string_incr(s, start=0):
 
 class Symbol(AtomicExpr, Boolean):
     """
+    Symbols are variables, and like all other variables, they need to
+    be assigned before you can use them.
+    SymPy :func:`Symbol` function returns just a single symbol taken from argument ``name`` whereas :func:`symbols` function can be used
+    to return a sequence of symbols.
+    Symbols can be used to build SymPy expressions which are immutable.
+
+    Parameters:
+    ===========
+
+    name: a string
+
     Assumptions:
-       commutative = True
+    ============
 
-    You can override the default assumptions in the constructor.
+    commutative = True
 
     Examples
     ========
 
-    >>> from sympy import symbols
+    >>> from sympy import Symbol, symbols
+    >>> a = Symbol('a')
+    >>> x, y, z = symbols('x y z')
+
+    SymPy will automatically pretty print symbols with greek letters.
+
+    >>> alpha, beta, gamma = symbols('alpha beta gamma')
```

Comment:
I guess this should be `alpha, beta, gamma = symbols('α  β γ')` for the doctest to pass

Reply:
No

Reply:
Sorry my bad , I wrongly understood what you were trying to implement!